lib/cadet_web/views/assessments_helpers.ex

```diff
@@ -140,7 +140,7 @@ defmodule CadetWeb.AssessmentsHelpers do
         answer: :answer,
         student_name: :student_name
       }),
-      "score",
+      "relative_score",
```

Comment:
will this be confusing with the score? Is this relative score the computed final score on the leaderboard? Maybe we can name it `final_score` or something along those line?

Reply:
ok